packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -525,55 +542,58 @@ export class AssetGraphBuilder {
 
   propagateSymbolsDown(
     visit: (
-      node: AssetNode,
+      nodeId: NodeId,
```

Comment:
Can you pass in the `AssetNode` as before here and retrieve the node in `propagateSymbolsDown ` (and not in the callback)?